### PR TITLE
Auto-trigger outerloop CI builds upon PR

### PR DESCRIPTION
**Pull request by iamjasonp**
* Change netci.groovy to auto-trigger all builds except code coverage on all builds rather than excluding outerloops as we currently do
* ~~Change netci.groovy to put PR queues into folders rather than one monolithic folder in dotnet-ci Jenkins dashboard~~
* Sync netci.groovy OS definitions to what we see in the dotnet/corefx repo

### CODE REVIEW

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
Note: I took out the "branchify" change that I intended to do; this needs to be separately coordinated.

**Comment by iamjasonp** [comment]
Ready for review

**Comment by StephenBonikowsky** [comment]
LGTM

File: netci.groovy [hunk 1]
```diff
@@ -1,41 +1,58 @@
...
 import jobs.generation.Utilities;
+import jobs.generation.JobReport;
 
+// The input project name (e.g. dotnet/corefx)
 def project = GithubProject
+// The input branch name (e.g. master)
+// We don't use this variable in this repo yet - we use branchName and loop across that
+def branch = GithubBranchName
+
+def projectFolderName = Utilities.getFolderName(project)
```
Comment (zhenlan) [inline netci.groovy]: Did not see this is used anywhere. To be used in next PR?

**Comment by zhenlan** [comment]
LGTM

**Pull request merged**